programming.md

```diff
@@ -1,29 +1,29 @@
 # Programming And Software Development
 
-This file contains resources to learn programming and software development. This file is divided into 2 parts - Goal-Oriented and General Purpose. 
+This file contains resources to learn programming and software development. This file is divided into 2 parts - Goal-Oriented and General Purpose.
 
-1. Goal-Oriented resources teach you to deliver an output like making a website or an app. 
+1. Goal-Oriented resources teach you to deliver an output like making a website or an app.
 2. General Purpose resources teach you a particular language like Python or JavaScript.
 
 A Goal-Oriented listing has higher priority so if a resource falls in both general purpose and goal-oriented, it automatically goes in goal-oriented and is not duplicated.
 
 **Goal-Oriented**
 
-* [Android Development](#android-development)
-* [Web Development](#web-development)
+- [Android Development](#android-development)
+- [Web Development](#web-development)
+- [Unity 3D](#unity-3d)
 
 **General Purpose**
 
-* [C++](#cpp)
-* [Django](#django)
-* [Git](#git)
-* [Java](#java)
-* [JUnit](#junit)
-* [JavaScript](#javascript)
-* [NodeJS](#nodejs)
-* [Python](#python)
-* Ruby
-
+- [C++](#cpp)
+- [Django](#django)
```

Comment:
This is a change that is not needed for this PR. I guess it was done by your code editor. Please remove it.